.travis.yml

```diff
@@ -1,7 +1,7 @@
 dist: bionic
 language: julia
 julia:
-  - 1.6.2
+  - 1.7.0
```

Comment:
what is rationale behind updating julia version? performance improvement?

Reply:
yes, but this is just Travis update